scripts/setup.py

```diff
@@ -78,10 +81,18 @@ def download_and_install_package(url_to_retrieve, filename):
             downloaded.
         filename: string. The name of the tar file.
     """
+
     python_utils.url_retrieve(url_to_retrieve, filename=filename)
-    tar = tarfile.open(name=filename)
-    tar.extractall(path=common.OPPIA_TOOLS_DIR)
-    tar.close()
+    if os.path.splitext(filename)[1] == '.zip' and common.is_windows_os():
```

Comment:
Does this function actually get called with .zip files? The old code just used tarfile.open().

Reply:
Yes, currently the only case is the node package on Windows. 

https://github.com/oppia/oppia/blob/cbce048808f146a5bc8d860a8b842b76754670f6/scripts/setup.py#L140

Reply:
I think that seems a little odd, given that the existing function is for tar files. Maybe make a new function that is single-purpose instead?

Reply:
Updated